core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1915,7 +1918,40 @@ public static int truncate(int v, int x) {
     }
     return v - remainder;
   }
+  public static String age(Long ts1, DataContext root) {
+    return age(ts1, null, root);
+  }
+  public static String age(Long ts1, Long ts2) {
```

Comment:
no real need in this method as you can always call `String age(Long ts1, Long ts2, DataContext root)`